app/controllers/controller.rb

```diff
@@ -0,0 +1,91 @@
+
+###### INDEX ######
+
+get '/' do
+  erb :home
+end
+
+# get '/hello' do
+#   erb :new
+# end
+
+get '/questions/new' do
+  # get form to add new question
+
+  erb :create_new_question
+end
+
+post '/questions' do
+  title = params[:title]
```

Comment:
no need to set these as a local variable if you are just going to use it once a few lines below.